vm-standalone-pvc-k8s/source.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: VolumeImportSource
+metadata:
+  name: cirros-source
+spec:
+  source:
+    registry:
+      # Image customized for ramen testing.
+      # TODO: Consume from ramendr repo when available.
+      url: "docker://quay.io/nirsof/cirros:0.6.2"
```

Comment:
This image includes now the ramen logging service, starting when the vm
starts and logging every 10 seconds to /var/log/ramen.log.

We can use this to track how the amount of data lost during failover and
the time when we the vm was down during relocate.

Example usage:

```
$ virtctl ssh cirros@sample-vm -n kubevirt-sample --context dr1 --command "tail -f /var/log/ramen.log"
Wed Oct 25 23:43:12 UTC 2023
Wed Oct 25 23:43:22 UTC 2023
Wed Oct 25 23:43:32 UTC 2023
Wed Oct 25 23:43:42 UTC 2023
```